pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -57,20 +76,24 @@ type MizarArktosNetworkController struct {
 }
 
 // NewMizarArktosNetworkController starts arktos network controller for mizar
-func NewMizarArktosNetworkController(netClientset *arktos.Clientset, kubeClientset *kubernetes.Clientset, networkInformer arktosinformer.NetworkInformer, grpcHost string, grpcAdaptor IGrpcAdaptor) *MizarArktosNetworkController {
+func NewMizarArktosNetworkController(vpcDefaultTemplatePath, subnetDefaultTemplatePath string, dynamicClient dynamic.Interface, discoveryClient discovery.DiscoveryInterface, netClientset *arktos.Clientset, kubeClientset *kubernetes.Clientset, networkInformer arktosinformer.NetworkInformer, grpcHost string, grpcAdaptor IGrpcAdaptor) *MizarArktosNetworkController {
```

Comment:
vpcDefaultTemplatePath and subnetDefaultTemplatePath can be generated by the init function of MizarArktosNetworkController. No need to pass in as argument

Reply:
yes, will make change for master branch.

Reply:
Done and updated in q131172019:CarlXie-poc-2022-01-30-tenant-vpc-support-final